### DISCUSSION TITLE
Jellyfin media player from PPA?

### DISCUSSION THREAD

**Discussion by pixel24** [jellyfin/jellyfin-desktop#442] [Q&A]
Hi@all,

i use Debian Linux on my desktop and have downloaded and installed the Debian package from the project page. Works great. Does anyone know a PPA for Ubuntu 22.04 ? Then future updates also come automatically to the system.

with best
pixel24

**Comment by matthewpritchett** [comment]
I ended up creating a bash script to auto-update it. 

https://github.com/matthewpritchett/htpc/blob/main/usr/local/bin/update-jellyfinmediaplayer.sh

And opened an issue to see if it could be included in the official Jellyfin repos.

https://github.com/jellyfin/jellyfin-media-player/issues/550
